feat: :sparkles: submission post to OIS conference 2026 (#190)
seedcase-project/community: posts/conference-ois-2026/submission.qmd

author:
  - name: Kristiane Beicher
    orcid: "https://orcid.org/0000-0001-7556-9566"
  - name: Signe K. Brødbæk
    orcid: "https://orcid.org/0009-0000-2208-7088"
  - name: Luke W. Johnston
    orcid: "https://orcid.org/0000-0003-4169-2616"
  - name: Joel Ostblom
    orcid: "https://orcid.org/0000-0003-0051-3239"
  - name: Marton Vago
    orcid: "https://orcid.org/0009-0007-4628-655X"
# Prevent it from entering the post listing page.
exclude-from-listing: true
---

::: content-hidden
From: https://ois-research-conference.org/ois-research-conference-2026/

Submission word limit: 250-500.

Words in document: 497

Submitting to track 1: Open and collaborative approaches along the
entire scientific research process, including opportunities, challenges,
and contingency factors. Possible topics include but are not limited to:

-   Open data/material sharing/re-use
-   Platforms facilitating open and collaborative approaches to
    scientific research
:::

Across much of science, we suffer from a lack of software and data
engineering resources, skills, and research-specific tools. This has

> negative impacts on our abilities to generate reliable results and be

Reviewer: ```suggestion
negative impacts on our ability to generate reliable results and be
```

productive and effective as researchers. Part of the issue is a lack of

> a general awareness by researchers, funding agencies, and organizations

Reviewer 2: ```suggestion
general awareness among researchers, funding agencies, and organizations
```

> on the importance of software and data engineering in research, as well

Reviewer: ```suggestion
general awareness among researchers, funding agencies, and organizations
about the importance of software and data engineering in research, as well
```
about or of

as a lack of systemic incentive structures to dedicate resources and

> time to these software and data practices. A major consequence is that

Reviewer 2: ```suggestion
time to these practices. A major consequence is that
```

we face many organizational and technical challenges with managing,
sharing, building, and using research data, not least of which is it

> impairs our ability to follow FAIR and open science practices.

Reviewer 3: ```suggestion
FAIR and open practices are beneficial for science as they encourage
reproducible, transparent, and shareable data practices.

While individual researchers sometimes are capable of implementing FAIR
and open practices on their own, the availability of software and data
engineering resources and tools greatly facilitates the adoption of
these practices.

Unfortunately, there is a lack of systemic structures to incentivize
researchers, organizations, and funding agencies to appreciate the
importance of software and data engineering in research contexts and to
dedicate sufficient resources to implementing them in practice.

A major consequence from this incentive misalignment is that researchers
trying to adhere to FAIR and open practices, face organizational and
technical challenges with managing, sharing, building, and using
research data, which in turn hampers scientific progress.
```

What do you think of this restructuring of the intro? (You can delete the spaces, I just included them here to make the structure clearer) I tried to organize according to the following:
 
1. Benefit of practices
2. Problem with implementing them
3. Cause of problem
4. Consequence of cause

Then in the next paragraph, we present our solution to (part) of this problem.

Through the Seedcase Project (\<seedcase-project.org\>), we aim to
resolve some of these issues. We are currently building a framework to

> make building and managing FAIR research data that is easier, modern,

Reviewer: And maybe align the adjectives so they are either all comparative ("more modern") or not ("easy")?

organized, and robust. We plan to achieve this by developing a series of
loosely coupled tools that work together within a specific framework.

> The tools we've published are:

Reviewer 3: ```suggestion
The tools we've published so far are:
```

-   Sprout (\<sprout.seedcase-project.org\>): A Python package that is
    designed to help create and manage a standardised and organised
    structure for storing and describing research data. Using modern
    data engineering and management practices, Sprout helps to ensure

>     and enforce that research data are well-designed, discoverable,

Reviewer 2: ```suggestion
    data engineering and management practices, Sprout promotes
    research data that are well designed, discoverable,
```

    well-documented, and ultimately more (re)usable for later analyses.
    This package is built around the Data Package standard
    (\<datapackage.org\>).
-   `check-datapackages` (\<check-datapackages.seedcase-project.org\>):

>     A Python package that checks your Data Package's metadata against

Reviewer: ```suggestion
    A Python package that checks a Data Package's metadata against
```
Maybe here "you" is a bit too informal, reads a bit too documentation-y?

>     the Data Package standard, to ensure that your metadata is compliant

Reviewer 2: ```suggestion
    the Data Package standard to ensure that a Data Package's metadata is compliant
```

    with the standard.
-   Template Data Package
    (\<template-data-package.seedcase-project.org\>): This is a template
    for creating a new Data Package following the Seedcase structure.

>     Use this template to quickly get started with a new Data Package

Reviewer: Maybe rewrite to be descriptive like the other items and not instructional?

    with all the necessary files and configurations in place, including
    for developing the Data Package.

The tools we are actively developing or will develop in the near future

> include:

Reviewer 3: ```suggestion
Tools that are planned or in early development:
```

I think using "active development" here makes it sound like the published tools are not actively developed, which they are.

-   Flower (\<flower.seedcase-project.org\>): A Python package and
    command-line tool to generate and display your `datapackage.json`

>     file in a human-friendly way. This is early in development.

Reviewer 3: ```suggestion
    file in a human-friendly way. In early development.
```

Save words

-   Propagate: A planned tool to make it easier to submit and process
    reproducible requests for a subset of data from a larger Data
    Package through a machine-readable set of instructions.
-   Garden: A planned tool to manage and track research projects using a
    Data Package

> Not only do we publish these data engineering tools, we also publish our

Reviewer: ```suggestion
As well as publishing these data engineering tools, we also publish our
```
Feels less marked to me and avoids repeating the same structure in the next paragraph

internal developer tools and documentation. Our roadmap at
\<seedcase-project.org/roadmap/\> lists all of our external and internal

> products and their development status.

Reviewer 3: ```suggestion
In addition to publishing data engineering tools, we also publish our
internal developer tools, documentation, and guides to effective
collaborative practices. Our roadmap at
\<seedcase-project.org/roadmap/\> lists all of our external and internal
products and their development status.
```

I think the teamwork/collab docs are also valuable artefacts from seedcase.

Ultimately, we work towards helping researchers do "better science in

> less time", not only those in academic settings in Denmark but also in

Reviewer: ```suggestion
less time", not only in academic settings in Denmark but also in
```

> industry and across the globe.

Reviewer 3: ```suggestion
In addition to publishing data engineering tools, we also publish our
internal developer tools, documentation, and guides to effective
collaborative practices. Through these publications, we contribute to
our ultimate goal of helping researchers do "better science in less
time", not only in an academic settings in Denmark but also in industry
and across the globe. Our progress towards this goal can be found in our
roadmap at \<seedcase-project.org/roadmap/\> which lists all of our
products and their development status.
```

Connecting these paragraphs a bit and add the the teamwork/collab docs which I think are also valuable artifacts from seedcase.
